tox/config.py

```diff
@@ -124,6 +136,20 @@ def postprocess(self, testenv_config, value):
             else:
                 name = depline.strip()
                 ixserver = None
+
+                # we need to process options, in case they contain a space,
+                # as the subprocess call to pip install will otherwise fail.
+
+                # in case of a short option, we remove the space
+                for option in PIP_INSTALL_SHORT_OPTIONS_ARGUMENT:
+                    if name.startswith(option):
+                        name = option + name[len(option):].strip()
+
+                # in case of a long option, we add an equal sign
+                for option in PIP_INSTALL_LONG_OPTIONS_ARGUMENT:
+                    if name.startswith(option + ' '):
+                        name = option + '=' + name[len(option):].strip()
```

Comment:
I think we should to use string format notation for this

Reply:
Good idea. Changed.